examples/user_input.rs

```diff
@@ -98,6 +93,21 @@ fn main() -> Result<(), Box<dyn Error>> {
                 .style(Style::default().fg(Color::Yellow))
                 .block(Block::default().borders(Borders::ALL).title("Input"));
             f.render_widget(input, chunks[1]);
+            match app.input_mode {
```

Comment:
I don't think we need two functions for this, smth like:

```rust
fn set_cursor(&mut self, pos: Option<(u16, u16)>) 
```
should be enough.

Reply:
I don't think it's obvious what `set_cursor(None)` means; `hide_cursor()` is way more descriptive. But you're the boss, so I'll change it :)

Reply:
Addressed by https://github.com/fdehau/tui-rs/pull/309/commits/af2c8b0ac33c9c154cef7fdf0999b5558751160e.